express/blocks/floating-button/floating-button.css

```diff
@@ -117,6 +117,19 @@ main > div:not(.banner-container) a.button.same-as-floating-button-CTA {
   display: none;
 }
 
+main [data-audience].floating-button-wrapper {
+  display: none;
+}
+
+body[data-device="mobile"] main .floating-button-wrapper[data-audience="mobile"][data-section-status="loaded"],
+body[data-device="desktop"] main .floating-button-wrapper[data-audience="desktop"][data-section-status="loaded"] {
```

Comment:
The reason for this specific rule is that the generic audience specific rules get overwritten by the `.floating-button-wrapper` specific rules. And `initial` actually sets them back to `block` instead of `flex`, which is defined by browser _user agent stylesheet_.